iac-extensions/spring-config/build.gradle.kts

```diff
@@ -15,3 +18,17 @@ dependencies {
     testImplementation(libs.sonar.analyzer.test.commons)
     testImplementation(testFixtures(project(":iac-common")))
 }
+
+tasks.generateGrammarSource {
+    maxHeapSize = "64m"
+    arguments = arguments + listOf("-visitor", "-long-messages", "-no-listener", "-package", "org.sonar.iac.springconfig.parser.properties")
+    // Due to bug in ANTLR Gradle plugin https://stackoverflow.com/a/49388412
+    outputDirectory =
+        project.layout.buildDirectory.dir(
+            "generated-src/antlr/main/org/sonar/iac/springconfig/parser/properties/"
```

Comment:
Can it be simple `generated-src/main` and not `generated-src/antlr/main`?

Reply:
It has to start with `generated-src/antlr/main` as this prefix is used as output to the task. Not following this convention requires additional configuration of this task or `sourcesJar`.